cli/plugin/src/main/java/io/helidon/build/cli/plugin/Plugin.java

```diff
@@ -56,6 +56,10 @@ public static void execute(String[] args, Consumer<String> logConsumer) throws E
      * Exception that is always logged with only the message.
      */
     public static class Failed extends Exception {
+
```

Comment:
Why do we need javadoc on private fields?

Reply:
Because the Javadoc tools complains with a warning that says "No comment". This is happening only in Exception classes. If there is public getter method for the field that warning goes away.